api/controllers/PromotionController.js

```diff
@@ -1,7 +1,9 @@
 var Promotion = require('../models/Promotion');
+var Restaurant = require('../models/Restaurant');
 var User = require('../models/User');
 var auth = require('../utils/auth');
 var csv = require('fast-csv');
+var superagent = require('superagent');
```

Comment:
this is a node.js native thing? no need to add to `package.json`?

Reply:
I guess we don't need to add it.
-- It works pretty well.

Reply:
I will add it to package.json anyway, just in case.

Reply:
Here is the screen shot I got when I run 
-- npm install superagent
![capture](https://user-images.githubusercontent.com/12376493/34088061-074da55a-e375-11e7-9748-255cf3a1dff9.PNG)

Seems that it is under supertest and swagger-express-mw.
-- And the package.json file is not changed at all.